src/AddressRegistry.Consumer.Read.StreetName/StreetNameBosaItemConsumer.cs

```diff
@@ -0,0 +1,50 @@
+namespace AddressRegistry.Consumer.Read.StreetName
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Autofac;
+    using Be.Vlaanderen.Basisregisters.MessageHandling.Kafka.Simple;
+    using Be.Vlaanderen.Basisregisters.ProjectionHandling.Connector;
+    using Projections;
+
+    public class StreetNameBosaItemConsumer
+    {
+        private readonly ILifetimeScope _container;
+        private readonly KafkaOptions _options;
+        private readonly StreetNameConsumerOptions _streetNameConsumerOptions;
+
+        public StreetNameBosaItemConsumer(
+            ILifetimeScope container,
+            KafkaOptions options,
+            StreetNameConsumerOptions streetNameConsumerOptions)
+        {
+            _container = container;
+            _options = options;
+            _streetNameConsumerOptions = streetNameConsumerOptions;
+        }
+
+        public Task Start(CancellationToken cancellationToken = default)
+        {
+            var bosaItemProjections = new ConnectedProjector<StreetNameConsumerContext>(Resolve.WhenEqualToHandlerMessageType(new StreetNameBosaItemProjections().Handlers));
+
+            var consumerGroupId = $"{nameof(AddressRegistry)}.{nameof(StreetNameLatestItemConsumer)}.{_streetNameConsumerOptions.Topic}{_streetNameConsumerOptions.ConsumerGroupSuffix}";
```

Comment:
consumerGroupId should be unique for bosa / latest. So maybe append after the suffix something unique like `.bosa`.